docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
```

Comment:
Can you please give as a hint how to make it work? I'm new to docker :(
I've tried:
`docker run --rm -v "$PWD":/usr/src/github.com/hudl/fargo -w /usr/src/github.com/hudl/fargo golang:1.6 go test ./...`

But it gives me :
```Error response from daemon: Invalid bind mount spec "/C/Users/damian.turczynski/Documents/hudl/go/src/github.com/hudl/fargo;C:\\Program Files\\Git\\usr\\src\\github.com\\hudl\\fargo": Invalid volume destination path: '\Program Files\Git\usr\src\github.com\hudl\fargo' mount path must be absolute..```

I love the idea though.